### ISSUE TITLE
Ctrl-click on a Gnx-based UNL gives errors

### ISSUE THREAD

**Issue by LewisNeal** [leo-editor/leo-editor#3458]
See PR #3640.

Ctrl-click on a Gnx-based UNL of a node which has @language directive
@language css
gives errors in Log pane:

is_sentinel: can not happen. delims: (None, '/*', '*/')
is_sentinel: can not happen. delims: (None, '/*', '*/')
is_sentinel: can not happen. delims: (None, '/*', '*/')
is_sentinel: can not happen. delims: (None, '/*', '*/')

**Comment by tbpassin** [comment]
In trying to repo this, I think I may have found the (or a) problem.

Here is the unl string I used for testing:
`unl:gnx://workbook.leo#tom.20230721114823.1`

This navigated correctly.  However, I accidentally typed `@language css` at the top of the node _that contained the UNL I was testing_.  When I deleted that line, suddenly the node became uncolored, the UNLs and GNXs on the page were not colored, and CTRL-Clicking failed with @LewisNeals's reported error messages.

After I navigated away from the node and returned, everything colored and acted normally.

To repeat, these changes involving the language directive occurred _in the dispatching node_, not the target node.

This behavior did not repeat the next time I tried it, but I definitely got it once.

```
Leo 6.7.4-devel, devel branch, build 5be2284604
2023-07-22 09:28:26 -0500
Python 3.11.4, PyQt version 6.4.3
Windows 10 AMD64 (build 10.0.19045) SP0
```
